tests/test_reply_headers.js

```diff
@@ -36,6 +36,76 @@ test('reply headers as function work', async t => {
   scope.done()
 })
 
+// Skipping these two test because of the inconsistencies around raw headers.
+// - they often receive the lower-cased versions of the keys
+// - the resulting order differs depending if overrides are provided to .reply directly or via a callback
+// - replacing values with function results isn't guaranteed to keep the correct order
+// - the resulting `headers` object itself is fine and these assertions pass
```

Comment:
Ooph

Reply:
This directly relates to the convo in the other thread about rawHeaders.